### ISSUE TITLE
Same output given be model irrespective of input.

### ISSUE THREAD

**Issue by debjyoti0891** [fchollet/keras#3378]
Hi,

I trained a model using keras and the validation accuracy is above 0.99 and validation loss is oabout 0.192. While training the model, the accuracy increases while the loss decreases, as expected. Even when testing the model, the results are good.
Test score: 0.0192234714976
Test accuracy: 0.995767716669

However, when I am trying to use the model to predict, I am always getting the same output irrespective of the input vectors. My dataset has around 6000 samples. Kindly help.

Here is the model:
# Layer (type)                     Output Shape          Param #     Connected to                     

embedding_1 (Embedding)          (None, 17, 1020)      122400      embedding_input_1[0][0]          

---

lstm_1 (LSTM)                    (None, 100)           448400      embedding_1[0][0]                

---

activation_1 (Activation)        (None, 100)           0           lstm_1[0][0]                     

---

dense_1 (Dense)                  (None, 2040)          206040      activation_1[0][0]  
===============================================================================Total params: 776840

---

Code for prediction:
predicted = model.predict_proba(X_test)
